### PR TITLE
Update

### PR DESCRIPTION
**Pull request by splitti**
1.You need another package also installed
2. Changing to new Varname

### CODE REVIEW

File: pi-blaster.c [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 #define _POSIX_C_SOURCE 200809L
 #define _XOPEN_SOURCE   700
-#define _BSD_SOURCE
+#define _DEFAULT_SOURCE
```
Comment (sarfata) [inline pi-blaster.c:35]: Can you explain this change?

Comment (splitti) [reply pi-blaster.c:35]: There where warning while compiling. This settings fixed it.

Comment (sarfata) [reply pi-blaster.c:35]: For future readers, this stack overflow answer gives a bit more details: https://stackoverflow.com/questions/29201515/what-does-d-default-source-do

**Comment by sarfata** [comment]
thanks @splitti

**Pull request merged**